livekit/attrs.go

```diff
@@ -0,0 +1,19 @@
+package livekit
+
+// Names of participant attributes for SIP.
+const (
+	// AttrSIPPrefix is shared for all SIP attributes.
+	AttrSIPPrefix = "lk.sip."
```

Comment:
do you think the `lk.` namespace adds value here? IMO since we don't have any reserved namespace, it could just be tagged SIP?

Reply:
Is there a reason for us not wanting a reserved namespace?

Reply:
the mechanism for us to set these attributes is via token grants, we do not have a way to enforce that a particular component is SIP vs something else. Since the developer is responsible for issuing tokens, they should be "trusting" information that they are providing.